Jellyfin.Api/Helpers/StreamingHelpers.cs

```diff
@@ -169,7 +169,7 @@ public static class StreamingHelpers
 
             string? containerInternal = Path.GetExtension(state.RequestedUrl);
 
-            if (string.IsNullOrEmpty(streamingRequest.Container))
```

Comment:
This might need to be `string.IsNullOrEmpty(containerInternal)` instead judging by the next one, but that depends on if we want the explicit setting to override this.

Reply:
Given the code, it looks to me as if streamRequest.Container should be used if defined, with containerInternal as a fallback.

the coding doesn't site write if it is (string.IsNullOrEmpty(streamingRequest.Container))) as surely you would set containerInternal to string.Empty ?

Reply:
I suppose that works, but I don't like the order all that much I guess. I assumed that the container would be specified more often than not, hence I felt the Path.GetExtension call would be done often without effect.